### PR TITLE
fix aggregate bug and cve problem

### PR DESCRIPTION
**Pull request by 2511689622**

### CODE REVIEW

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

**Comment by shishupei** [comment]
<table style="border-collapse: collapse">
    <tr>
        <th>检查项</th>
        <th>状态</th>
        <th>日志</th>
    </tr>

<tr>
    <td>敏感信息扫描</td>
    <td>&#9989;</td>
    <td></td>
</tr>

<tr>
    <td>开源代码漏洞扫描</td>
    <td>&#10060;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_开源代码漏洞扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>Check代码检查</td>
    <td>&#10060;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_Check代码检查.html">查看日志</a></td>
</tr>

<tr>
    <td>DT覆盖率</td>
    <td>0.00%</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_dt覆盖率.html">查看日志</a></td>
</tr>

<tr>
    <td>流水线链接</td>
    <td colspan="2"><a href="https://devcloud.cn-north-4.huaweicloud.com/cicd/project/3a76c1785dda4b13a399937d1978f240/pipeline/detail/99c7d59e96c74d5c963cbf39a9049518/abd15315d2c642ae9f2266731421b444">点击跳转</a></td>
</tr>
</table>

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

**Comment by shishupei** [comment]
<table style="border-collapse: collapse">
    <tr>
        <th>检查项</th>
        <th>状态</th>
        <th>日志</th>
    </tr>

<tr>
    <td>敏感信息扫描</td>
    <td>&#9989;</td>
    <td></td>
</tr>

<tr>
    <td>开源代码漏洞扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_开源代码漏洞扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>Check代码检查</td>
    <td>&#10060;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_Check代码检查.html">查看日志</a></td>
</tr>

<tr>
    <td>DT覆盖率</td>
    <td>0.00%</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_dt覆盖率.html">查看日志</a></td>
</tr>

<tr>
    <td>流水线链接</td>
    <td colspan="2"><a href="https://devcloud.cn-north-4.huaweicloud.com/cicd/project/3a76c1785dda4b13a399937d1978f240/pipeline/detail/99c7d59e96c74d5c963cbf39a9049518/b67600b3911248df96327d2696cef917">点击跳转</a></td>
</tr>
</table>

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

**Comment by shishupei** [comment]
<table style="border-collapse: collapse">
    <tr>
        <th>检查项</th>
        <th>状态</th>
        <th>日志</th>
    </tr>

<tr>
    <td>敏感信息扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_敏感信息扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>开源代码漏洞扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_开源代码漏洞扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>DT覆盖率</td>
    <td>0.00%</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_dt覆盖率.html">查看日志</a></td>
</tr>

<tr>
    <td>流水线链接</td>
    <td colspan="2"><a href="https://devcloud.cn-north-4.huaweicloud.com/cicd/project/3a76c1785dda4b13a399937d1978f240/pipeline/detail/99c7d59e96c74d5c963cbf39a9049518/7c6646590cbe446f8a17af77d3a849ff">点击跳转</a></td>
</tr>
</table>

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

**Comment by shishupei** [comment]
<table style="border-collapse: collapse">
    <tr>
        <th>检查项</th>
        <th>状态</th>
        <th>日志</th>
    </tr>

<tr>
    <td>敏感信息扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_敏感信息扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>开源代码漏洞扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_开源代码漏洞扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>DT覆盖率</td>
    <td>0.00%</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_dt覆盖率.html">查看日志</a></td>
</tr>

<tr>
    <td>Check代码检查</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_Check代码检查.html">查看日志</a></td>
</tr>

<tr>
    <td>流水线链接</td>
    <td colspan="2"><a href="https://devcloud.cn-north-4.huaweicloud.com/cicd/project/3a76c1785dda4b13a399937d1978f240/pipeline/detail/99c7d59e96c74d5c963cbf39a9049518/b6c63cf01fac48c4ba82814175c7b7e9">点击跳转</a></td>
</tr>
</table>

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

**Comment by shishupei** [comment]
<table style="border-collapse: collapse">
    <tr>
        <th>检查项</th>
        <th>状态</th>
        <th>日志</th>
    </tr>

<tr>
    <td>敏感信息扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_敏感信息扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>开源代码漏洞扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_开源代码漏洞扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>DT覆盖率</td>
    <td>0.00%</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_dt覆盖率.html">查看日志</a></td>
</tr>

<tr>
    <td>Check代码检查</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_Check代码检查.html">查看日志</a></td>
</tr>

<tr>
    <td>安全编码扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_安全编码扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>流水线链接</td>
    <td colspan="2"><a href="https://devcloud.cn-north-4.huaweicloud.com/cicd/project/3a76c1785dda4b13a399937d1978f240/pipeline/detail/99c7d59e96c74d5c963cbf39a9049518/4d9a43d8df644534b187a0530a030f4a">点击跳转</a></td>
</tr>
</table>

File: src/main/java/com/easysoftware/infrastructure/fieldpkg/FieldPkgGatewayImpl.java [hunk 1]
```diff
@@ -65,12 +66,12 @@ public Map<String, Object> queryMenuByPage(final FieldPkgSearchCondition conditi
 
         IPage<FieldPkgDO> resPage = mapper.selectPage(page, wrapper);
         List<FieldPkgDO> list = resPage.getRecords();
-        List<FieldPkgVo> voList = FieldPkgConverter.toVo(list);
         long total = resPage.getTotal();
+        List<FieldPkgVo> voList = FieldPkgConverter.toVo(list);
 
         if (condition.getOs() == null && condition.getArch() == null && condition.getCategory() == null
                 && condition.getNameOrder() == null) {
-            voList = aggregateList(voList, condition, wrapper);
```
Comment (ZWJason) [inline src/main/java/com/easysoftware/infrastructure/fieldpkg/FieldPkgGatewayImpl.java:73]: 确认resPage.getTotal()返回最大数目， 统一变量类型

**Review by ZWJason — COMMENTED** [review]
确认resPage.getTotal()返回最大数目， 统一变量类型

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

**Comment by shishupei** [comment]
<table style="border-collapse: collapse" border=1>
    <tr>
        <th>检查项</th>
        <th>状态</th>
        <th>日志</th>
    </tr>

<tr>
    <td>敏感信息扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_敏感信息扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>开源代码漏洞扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_开源代码漏洞扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>DT覆盖率</td>
    <td>0.247146%</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_dt覆盖率.html">查看日志</a></td>
</tr>

<tr>
    <td>Check代码检查</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_Check代码检查.html">查看日志</a></td>
</tr>

<tr>
    <td>安全编码扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_安全编码扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>流水线链接</td>
    <td colspan="2"><a href="https://devcloud.cn-north-4.huaweicloud.com/cicd/project/3a76c1785dda4b13a399937d1978f240/pipeline/detail/99c7d59e96c74d5c963cbf39a9049518/8dd4e4797b8741949f3c40ecffc491e7">点击跳转</a></td>
</tr>
</table>

**Comment by shishupei** [comment]
<table style="border-collapse: collapse" border=1>
    <tr>
        <th>检查项</th>
        <th>状态</th>
        <th>日志</th>
    </tr>

<tr>
    <td>敏感信息扫描</td>
    <td>&#10060;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_敏感信息扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>开源代码漏洞扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_开源代码漏洞扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>DT覆盖率</td>
    <td>0.247146%</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_dt覆盖率.html">查看日志</a></td>
</tr>

<tr>
    <td>Check代码检查</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_Check代码检查.html">查看日志</a></td>
</tr>

<tr>
    <td>安全编码扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_安全编码扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>流水线链接</td>
    <td colspan="2"><a href="https://devcloud.cn-north-4.huaweicloud.com/cicd/project/3a76c1785dda4b13a399937d1978f240/pipeline/detail/99c7d59e96c74d5c963cbf39a9049518/97507e19343c4b519a1f4a09521644ea">点击跳转</a></td>
</tr>
</table>

File: src/main/java/com/easysoftware/infrastructure/fieldpkg/converter/FieldPkgConverter.java [hunk 2]
```diff
@@ -65,21 +64,13 @@ public static List<FieldPkgVo> toVo(final List<FieldPkgDO> doList) {
      */
     public static List<FieldPkgVo> aggregateVoByName(final List<FieldPkgVo> doList) {
```
Comment (shishupei) [inline src/main/java/com/easysoftware/infrastructure/fieldpkg/converter/FieldPkgConverter.java:65]: 注意格式

**Review by shishupei — APPROVED** [review]
同意合入

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

**Comment by shishupei** [comment]
<table style="border-collapse: collapse" border=1>
    <tr>
        <th>检查项</th>
        <th>状态</th>
        <th>日志</th>
    </tr>

<tr>
    <td>敏感信息扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_敏感信息扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>开源代码漏洞扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_开源代码漏洞扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>DT覆盖率</td>
    <td>0.247146%</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_dt覆盖率.html">查看日志</a></td>
</tr>

<tr>
    <td>Check代码检查</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_Check代码检查.html">查看日志</a></td>
</tr>

<tr>
    <td>安全编码扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySoftwareService/68/68_安全编码扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>流水线链接</td>
    <td colspan="2"><a href="https://devcloud.cn-north-4.huaweicloud.com/cicd/project/3a76c1785dda4b13a399937d1978f240/pipeline/detail/99c7d59e96c74d5c963cbf39a9049518/76b8b8ca15834172853c747abbba3b04">点击跳转</a></td>
</tr>
</table>

**Review by zhongjun2 — APPROVED** [review]

**Pull request merged**